daras_ai_v2/azure_doc_extract.py

```diff
@@ -77,6 +89,18 @@ def azure_form_recognizer(url: str, model_id: str, params: dict = None):
             case _:
                 sleep(1)
 
+def extract_records_prebuilt_read(result: dict, page_num: int) -> list[dict]:
```

Comment:
instead of creating a new func, update extract_records to handle both cases - eg I think its because the result won't have tables, so do `result.get('tables', [])` instead of `result['tables']`